### PR TITLE
Feature adjustable fonts

### PR DESCRIPTION
**Pull request by alxhoff**
Second attempt

### CODE REVIEW

**Comment by alxhoff** [comment]
@PhilippvK  I would just avoid committing at the moment so that when I fix this and rebase I can have the logs look like how they look in my video to minimize confusion for the students. If that makes sense.

File: lib/Gfx/TUM_Draw.c [hunk 1]
```diff
@@ -774,6 +766,9 @@ int vBindDrawing(void) // Should be called from the Drawing Thread
 		goto err_make_current;
 	}
 
+    if(renderer)
```
Comment (alxhoff) [inline lib/Gfx/TUM_Draw.c]: @PhilippvK 

This popped up with valgrind. Been so long since I looked into how SDL uses renderers, does this look correct?

**Comment by PhilippvK** [comment]
Let me guess that figuring this out was not that enjoyable?

**Comment by alxhoff** [comment]
hahaha for some reason fixing memory leaks is super satisfying. Also with Valgrind it's not too bad.

**Review by alxhoff — COMMENTED** [review]
> Let me guess that figuring this out was not that enjoyable?

**Comment by alxhoff** [comment]
I think it is fixed. Now to do some serious rebasing

File: lib/Gfx/TUM_Draw.c (hunk 1 above)
Comment (PhilippvK) [reply lib/Gfx/TUM_Draw.c]: I can have another look at it. The problem is, that there is almost no example code on the web, where thoses handles are cleaned up completely.

**Comment by PhilippvK** [comment]
> hahaha for some reason fixing memory leaks is super satisfying. Also with Valgrind it's not too bad.

So spending time adding that feature (valgrind) was actually worth it!

**Comment by PhilippvK** [comment]
Regarding the code review I can say that it works for me like before and the MQ stuff looks normal.

**Comment by alxhoff** [comment]
looks like I didn't ruin the rebase this time. master is no longer a mess either

**Comment by alxhoff** [comment]
I also think I solved #40 in doing so woop woop.

**Comment by PhilippvK** [comment]
Wow!

**Pull request merged**